frontend/src/core/cells/cells.ts

```diff
@@ -241,6 +242,10 @@ const {
       }
     }
 
+    const isMarkdown = newCellCode
+      ? LanguageAdapters.markdown().isSupported(newCellCode)
```

Comment:
isSupported returns true for empty string `""`., maybe we could check its not empty as well, or add a function `isMarkdown`